float64[] latitude
float64[] longitude
UTM[] utm_coordinates